Composer/packages/client/src/pages/botProject/SkillHostEndPoint.tsx

```diff
@@ -70,7 +70,14 @@ export const SkillHostEndPoint: React.FC<SkillHostEndPointProps> = (props) => {
   const { skillHostEndpoint } = useRecoilValue(settingsState(projectId));
 
   return (
-    <CollapsableWrapper title={formatMessage('Skill host endpoint')} titleStyle={title}>
+    <Fragment>
+      <div css={title}>{formatMessage('Call skills')}</div>
+      <div css={subtext}>
+        {formatMessage('TODO ADD COPY HERE. ')}
```

Comment:
Looks like you have something todo here

Reply:
Trailing spcae

Reply:
Getting text from design today, figma was missing it

Reply:
I have added text temporarily. There is an all up task to update copy across provisioning which will be handled in its own PR.